admin/pre_reg/views.py

```diff
@@ -102,10 +102,9 @@ class DraftDetailView(PreregAdmin, DetailView):
     context_object_name = 'draft'
 
     def get_object(self, queryset=None):
+        draft = DraftRegistration.load(self.kwargs.get('draft_pk'))
```

Comment:
Without a default, `.get` can raise an `AttributeError`, which is probably why it was previously all being done in the `try` block.